src/pages/RP_Resources.page

```diff
@@ -1,13 +1,13 @@
-<apex:page showHeader="true" sidebar="false" standardStylesheets="false" applyBodyTag="false">
+<apex:page showHeader="true" sidebar="false" standardStylesheets="false" applyBodyTag="false" lightningStyleSheets="true">
```

Comment:
Didn't catch this before but is there any reason to enable lightningStyleSheets here? I thought that was for automated styling with non sldsified pages?

Reply:
Not that I know of, I think I was just confused by this documentation: https://www.lightningdesignsystem.com/platforms/visualforce/

I can remove the the lightningStyleSheets attribute, it seems like it's redundant when apex:slds is used